tests/python/PyDocStyleBearTest.py

```diff
@@ -23,7 +23,44 @@ def hello():
 
 '''.splitlines(keepends=True)
 
-PyDocStyleBearTest = verify_local_bear(PyDocStyleBear,
-                                       valid_files=(good_file,),
-                                       invalid_files=(bad_file,),
-                                       tempfile_kwargs={"suffix": ".py"})
+PyDocStyleBearTest = verify_local_bear(
+    PyDocStyleBear,
+    valid_files=(good_file,),
+    invalid_files=(bad_file,),
+    tempfile_kwargs={"suffix": ".py"})
+
+PyDocStyleBearIgnoreArgTest = verify_local_bear(
+    PyDocStyleBear,
+    valid_files=(good_file, bad_file,),
+    invalid_files=(),
+    settings={'pydocstyle_ignore': 'D400, D200, D103'},
+    tempfile_kwargs={"suffix": ".py"})
+
+PyDocStyleBearIgnoreArgTest1 = verify_local_bear(
+    PyDocStyleBear,
+    valid_files=(good_file,),
+    invalid_files=(bad_file,),
+    settings={'pydocstyle_ignore': 'D400, D200'},
+    tempfile_kwargs={"suffix": ".py"})
```

Comment:
@sils1297 
* `PyDocStyleBearIgnoreArgTest1` checks if an invalid file yields results when only `some` errors are ignored.
* `PyDocStyleBearSelectArgTest` checks if an invalid file yields results when a particular error is `selected`.
* `PyDocStyleBearSelectArgTest1` checks if an invalid file yields no result when the selected error does not `occur`.

You sure I should remove these tests?